### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by Christoph-Wagner**

### CODE REVIEW

File: packages/typescriptlang-org/src/copy/en/index.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export const indexCopy = {
 
   index_dts: "Types On Every Desk",
   index_dts_copy: `
-<p>Most of the worlds JavaScript’s is un-typed, and inference can only go so far. To address this, the TypeScript team helps maintain <dt>Definitely Typed</dt> - a community project to provide types and inline documentation to existing JavaScript.
+<p>Most of the worlds JavaScript is un-typed, and inference can only go so far. To address this, the TypeScript team helps maintain <dt>Definitely Typed</dt> - a community project to provide types and inline documentation to existing JavaScript.
```
Comment (DanielRosenwasser) [inline packages/typescriptlang-org/src/copy/en/index.ts]:
```suggestion
<p>Most of the world's JavaScript is un-typed, and inference can only go so far. To address this, the TypeScript team helps maintain <dt>Definitely Typed</dt> - a community project to provide types and inline documentation to existing JavaScript.
```

Comment (DanielRosenwasser) [reply packages/typescriptlang-org/src/copy/en/index.ts]: @orta, does the site use smart or simple apostrophes throughout?

**Comment by orta** [comment]
Thanks - good catch

File: packages/typescriptlang-org/src/copy/en/index.ts (hunk 1 above)
Comment (orta) [reply packages/typescriptlang-org/src/copy/en/index.ts]: simple

**Pull request merged**